supporting_content/metrics/listunitprice.md

```diff
@@ -6,72 +6,85 @@ Current column mappings found in available data sets:
 
 | Provider  | Data set                                           | Column                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
 | :-------- | :------------------------------------------------- | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| AWS       | CUR;<br>AWS Price List API                         | pricing/publicOnDemandRate: The public On-Demand Instance rate in this billing period for the specific row of usage. If you have SKUs with multiple On-Demand public rates, the equivalent rate for the highest tier is displayed. For example, services offering free-tiers or tiered pricing.<br>_Note: Since currently only pricing/publicOnDemandRate definition includes note regarding highest tier, I'm not sure if volume-based pricing applies to all unit prices (rates) in CUR files. If it's not consistent, we might have to rely on the AWS Price List API._ |
+| AWS       | CUR;<br>AWS Price List API                         | the closest match - pricing/publicOnDemandRate: The public On-Demand Instance rate in this billing period for the specific row of usage. If you have SKUs with multiple On-Demand public rates, the equivalent rate for the highest tier is displayed. For example, services offering free-tiers or tiered pricing.<br>_Note: Since currently only pricing/publicOnDemandRate definition includes note regarding highest tier, I'm not sure if volume-based pricing applies to all unit prices (rates) in CUR files. If it's not consistent, we might have to rely on the AWS Price List API._ |
```

Comment:
I'm not sure how formal/informal we can be in these supporting content docs -- but the use of first and second person here (i.e. "I" and "you") feels off to me.  I would suggest using whatever convention we have for this -- and, if we lack any such convention, sidestepping any use of first and second person with some combination of nouns and passive voice would be preferable to me.

```suggestion
| AWS       | CUR;<br>AWS Price List API                         | the closest match - pricing/publicOnDemandRate: The public On-Demand Instance rate in this billing period for the specific row of usage. If there are SKUs with multiple On-Demand public rates, the equivalent rate for the highest tier is displayed. For example, services offering free-tiers or tiered pricing.<br>_Note: Since currently only pricing/publicOnDemandRate definition includes note regarding highest tier, it is unclear whether volume-based pricing applies to all unit prices (rates) in CUR files. If it's not consistent, the practitioner might have to rely on the AWS Price List API._ |
```